test/npb/ft/npadmana/DistributedFFT.chpl

```diff
@@ -36,6 +36,7 @@ prototype module DistributedFFT {
 
   public use BlockDist;
   public use FFTW;
+  import FFTW; // also allow FFTW.execute, etc
```

Comment:
Would just `public use FFTW as FFTW;` do the same thing here?  (I find both patterns slightly off-putting, but maybe that one slightly less so).  Is there any chance that `import FFTW` is all that's required?  That there aren't any unqualified uses of FFTW?  (seems unlikely).

Reply:
It looks like only `test/npb/ft/npadmana/ft_transposed.chpl` tests this module. That test just has `use DistributedFFT`. IIRC there were some FFTW symbols that it was expecting to have available also.

And yes we could do `public use FFTW as FFTW;` here.